CrazyflieEntry crazyflieEntry
uint32 mode #insert/update, remove
---

